libs/base/Data/Maybe.idr

```diff
@@ -60,3 +60,10 @@ lowerMaybe (Just x) = x
 export
 raiseToMaybe : (Monoid a, Eq a) => a -> Maybe a
 raiseToMaybe x = if x == neutral then Nothing else Just x
+
+||| Unwraps the first argument in case of `Just`
+||| returns its second argument otherwise
+export
+orElse : Maybe a -> Lazy a -> a
```

Comment:
This is `flip fromMaybe`. I am not sure we should add such variants of existing functions. :/

Reply:
Oops, it is. I absolutely agree. What about dup ?

Reply:
`dup` is a welcomed addition!